text/0000-ternary-conversion.md

```diff
@@ -0,0 +1,42 @@
++ Feature name: `ternary-conversion`
```

Comment:
```suggestion
+ Feature name: `ternary-numeric-conversion`
```
I feel like just conversion is a bit vague.

Reply:
I'll need to rename the file, will do